Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Base on offical Node.js Alpine image
```

Comment:
```suggestion
# Base image, for installing the dependencies.
```

Reply:
So, I decided to detail every stage. I'd be like this. What do you think? 🤔💭 
```suggestion
# Installation stage
# Use Node.js Alpine base image
```